Analysis/mk/makefile

```diff
@@ -82,7 +82,7 @@ USERINCLUDES += -isystem $(PYSYS)
 
 CXX=g++
 LD=g++
-CXXFLAGS= -Wall -Wextra -Werror -O2 -std=c++0x
+CXXFLAGS= -Wall -Wextra -Werror -O2 -std=c++0x -g
```

Comment:
when you get a seg fault it just tells you the line it seg faulted on which is useful for debugging